ts/screens/wallet/AddPaymentMethodScreen.tsx

```diff
@@ -147,7 +147,7 @@ export class AddPaymentMethodScreen extends React.Component<Props, State> {
                   </Grid>
                 </Left>
                 <Right style={AddMethodStyle.centeredContents}>
-                  <Icon
+                  <IconFont
                     name={itemInfo.item.icon}
                     color={variables.brandPrimary}
```

Comment:
Does this color (and also size) property take precedence over the default value of the theme?

Reply:
no they doesn't

Reply:
So what are they useful for?

Reply:
they was converted before connection with the style, i will fix it in the next commit